Indicators/InverseFisherTransform.cs

```diff
@@ -0,0 +1,124 @@
+    ///  
+    /// It is useful because it normalizes the current input, such as price, to fit within the range of the 
+    /// High and Low of the data window.
+    /// 
+    /// For a discussion of how it is used in trading to clarify other indicators turning points
+    /// see http://www.tradingsystemlab.com/files/The%20Inverse%20Fisher%20Transform.pdf
+    /// 
+    /// The formula comes from one of the activation functions of a neural network 
+    /// 
+    /// </summary>
+    public class InverseFisherTransform : WindowIndicator<IndicatorDataPoint>
+    {
+
+        private readonly Minimum _minLow;
+        private readonly Maximum _maxHigh;
+        private RollingWindow<IndicatorDataPoint> value1;
+        
+
+
+        /// <summary>
+        /// An Inverse Fisher Transform of Prices
+        /// </summary>
+        /// <param name="name">string - the name of the indicator</param>
+        /// <param name="period">The number of periods for the indicator window</param>
+        public InverseFisherTransform(string name, int period)
+            : base(name, period)
+        {
+            // Initialize the local variables
+            value1 = new RollingWindow<IndicatorDataPoint>(period);
+
+            // add two minimum values to the value1 to get things started
+            value1.Add(new IndicatorDataPoint(DateTime.MinValue, .0001m));
+            value1.Add(new IndicatorDataPoint(DateTime.MinValue, .0001m));
+
+            // Initialize the local variables
+            _maxHigh = new Maximum("MaxHigh", period);
+            _minLow = new Minimum("MinLow", period);
+        }
+
+        /// <summary>
+        ///     Initializes a new instance of the FisherTransform class with the default name and period
+        /// </summary>
+        /// <param name="period">The period of the WMA</param>
+        public InverseFisherTransform(int period)
+            : this("Fish_" + period, period)
+        {
+        }
+        /// <summary>
+        /// Computes the next value in the transform. 
+        /// value1 is a function used to normalize price withing the last _period day range.
+        /// value1 is centered on its midpoint and then doubled so that value1 wil swing between -1 and +1.  
+        /// value1 is also smoothed with an exponential moving average whose alpha is 0.33.  
+        /// 
+        /// Since the smoothing may allow value1 to exceed the _period day price range, limits are introduced to 
+        /// preclude the transform from blowing up by having an input larger than unity.
+        /// </summary>
+        /// <param name="window">The IReadOnlyWindow of Indicator Data Points for the history of this indicator</param>
+        /// <param name="input">IndicatorDataPoint - the time and value of the next price</param>
+        /// <returns></returns>
+        protected override decimal ComputeNextValue(IReadOnlyWindow<IndicatorDataPoint> window, IndicatorDataPoint input)
```

Comment:
do we use the 'window' parameter? If not maybe this class shouldn't derived from WindowIndicator